recipes/libhugetlbfs/build.sh

```diff
@@ -0,0 +1,18 @@
+if [[ ${ARCH} == "64" ]]; then
+    export ARCH=x86_64
+fi
+
+# Remove the static library from being installed
+sed -i 's/libhugetlbfs.a//g' Makefile
+
+# We only bulid native, so we want this to go to the correct location
+# The location is lib, nomatter what
+export LIB64=lib
+export LIB32=lib
+
+export BUILDTYPE=NATIVEONLY
+# Make library and tools, but not tests
+make libs tools -j${CPU_COUNT}
+
+# Do not install man pages
+make install-libs install-bin
```

Comment:
Do we need to specify `PREFIX` with either of these? Even if not, it might be good to include it (and `ARCH` depending on previous comment) for clarity so future readers know the `Makefile` is using these.

Reply:
done.